data/migrations/20190404190727_recipe_ingredients_table.js

```diff
@@ -0,0 +1,31 @@
+
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable('recipe-ingredients', tbl => {
+        tbl.increments()
+
+        tbl
+        .integer('recipe_id')
+        .unsigned()
+        .references('id')
+        .inTable('recipe')
+        .onDelete('CASCADE')
+        .onUpdate('CASCADE')
+
+        tbl
+        .integer('ingredients_id')
+        .unsigned()
+        .references('id')
+        .inTable('ingredients')
+        .onDelete('CASCADE')
+        .onUpdate('CASCADE')
+
+        tbl
+        .integer('ingredients_quantity')
```

Comment:
This should have been a floating number.

```suggestion
        .float('ingredients_quantity')
```